### ISSUE TITLE
No method `scale' for Origami::ContentStream

### ISSUE THREAD

**Issue by jizak** [gdelugre/origami#10]
I am trying in the console wiki example:
```
pdf = PDF.new

pdf.append_page
pdf.pages.first.write "Hello", size: 30
```
After executing the 3rd line I am getting an error:

```
No method `scale' for Origami::ContentStream
from /Users/pavlo/.rvm/gems/ruby-2.3.1@gk_4.2/gems/origami-2.0.1/lib/origami/object.rb:98:in `method_missing'
	from /Users/pavlo/.rvm/gems/ruby-2.3.1@gk_4.2/gems/origami-2.0.1/lib/origami/graphics/xobject.rb:295:in `set_text_scale'
	from /Users/pavlo/.rvm/gems/ruby-2.3.1@gk_4.2/gems/origami-2.0.1/lib/origami/graphics/xobject.rb:224:in `write'
	from /Users/pavlo/.rvm/gems/ruby-2.3.1@gk_4.2/gems/origami-2.0.1/lib/origami/graphics/xobject.rb:485:in `write'
	from (irb):14
	from /Users/pavlo/.rvm/gems/ruby-2.3.1@gk_4.2/gems/railties-4.2.7/lib/rails/commands/console.rb:110:in `start'
	from /Users/pavlo/.rvm/gems/ruby-2.3.1@gk_4.2/gems/railties-4.2.7/lib/rails/commands/console.rb:9:in `start'
	from /Users/pavlo/.rvm/gems/ruby-2.3.1@gk_4.2/gems/railties-4.2.7/lib/rails/commands/commands_tasks.rb:68:in `console'
	from /Users/pavlo/.rvm/gems/ruby-2.3.1@gk_4.2/gems/railties-4.2.7/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
	from /Users/pavlo/.rvm/gems/ruby-2.3.1@gk_4.2/gems/railties-4.2.7/lib/rails/commands.rb:17:in `<top (required)>'
	from bin/rails:4:in `require'
	from bin/rails:4:in `<main>'
```

**Comment by gdelugre** [comment]
This issue had already reported and fixed at #5, so it is actually already fixed on the git repository. However I didn't generate a new gem for this regression so I am going to do that.

**Comment by gdelugre** [comment]
Should now be fixed in the gem version ``2.0.2``.
